core/src/components/textarea/test/slot/index.html

```diff
@@ -51,51 +51,184 @@
       <ion-content id="content" class="ion-padding">
         <div class="grid">
           <div class="grid-item">
-            <h2>No Fill / Start</h2>
+            <h2>No Fill / Start Label</h2>
```

Comment:
I know in the select/input PRs we discussed using an icon instead of a button in the start slot since that's not a very common pattern. Can we do the same here for the a11y and slot files for `textarea`?

Reply:
Updated for input and textarea in https://github.com/ionic-team/ionic-framework/pull/28583/commits/ff3c9c80ba08ce6132be20391eaf8e7a4c25b1db 👍 I also added some headers to the `slot` tests to make them easier to parse.